### PR TITLE
Updating deploy profile operation to be interactive.

### PR DESCRIPTION
**Pull request by danielreed**
Covers issue #73 and also #89

### CODE REVIEW

File: src/listener/server-profile-templates.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ class ServerProfileTemplateListener extends Listener {
     this.respond(this.LIST_DEPLOYED_PROFILES, this.GetDeployedProfiles.bind(this));
     this.capabilities.push(this.BULLET + "Show profile(s) using a server profile template (e.g. show profiles using docker swarm).");
 
-    this.CREATE_PROFILES_FROM_TEMPATE = /(?:deploy|create) (:<count>\d+) profile[s]{0,1} (?:from|for|using) (:<host>.*?)(?:\/rest\/server-profile-templates\/)(:<templateId>[a-zA-Z0-9_-]*?)\.$/i;
+    this.CREATE_PROFILES_FROM_TEMPATE = /(?:deploy|create) a profile (?:from|for|using) (:<host>.*?)(?:\/rest\/server-profile-templates\/)(:<templateId>[a-zA-Z0-9_-]*?)\.$/i;
```
Comment (llvieira) [inline src/listener/server-profile-templates.js]: I would just rename this variable to `CREATE_PROFILE_FROM_TEMPATE`.

File: src/listener/server-profile-templates.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ class ServerProfileTemplateListener extends Listener {
...
     this.respond(this.CREATE_PROFILES_FROM_TEMPATE, this.DeployProfiles.bind(this));
     this.capabilities.push(this.BULLET + "Create profile(s) using a server profile template (e.g. create 1 profile from docker swarm).");
```
Comment (llvieira) [inline src/listener/server-profile-templates.js]: `Create profile` instead of `Create profile(s)`.

Comment (llvieira) [reply src/listener/server-profile-templates.js]: (e.g. create **a** profile from docker swarm).

File: src/listener/server-profile-templates.js (hunk 1 above)
Comment (danielreed) [reply src/listener/server-profile-templates.js]: Good idea

File: src/listener/server-profile-templates.js (hunk 2 above)
Comment (danielreed) [reply src/listener/server-profile-templates.js]: Good catch thanks!

**Comment by llvieira** [comment]
@danielreed now everything looks great. It can already be merged, right?

**Comment by danielreed** [comment]
@llvieira yes i think so, I'll go ahead and merge.  Appreciate the review feedback.

**Pull request merged**